core/roboconf-dm/src/main/java/net/roboconf/dm/internal/api/impl/TargetsMngrImpl.java

```diff
@@ -442,7 +454,8 @@ public String findTargetId( AbstractApplication app, String instancePath ) {
 		if( targetDir.isDirectory()){
 			List<File> scriptFiles = Utils.listAllFiles(targetDir);
 			for( File scriptFile : scriptFiles) {
-				if( scriptFile.getName().contains(Constants.SCOPED_SCRIPT_SUFFIX)) {
+				String name = scriptFile.getName();
+				if( !(name.contains(Constants.FILE_EXT_PROPERTIES) || name.contains(".tmp") || name.contains(CREATED_BY)) ) {
```

Comment:
A ".tmp" file is sometimes created  during some unit tests. So it must be ignored during the copy. 
 But by adding the prefix test condition, it's not necessary. I remove it.